### DISCUSSION TITLE
better error messages

### DISCUSSION THREAD

**Discussion by Niklas-Mages** [FreeRDP/FreeRDP#9202] [Ideas]
It would be nice if it gives back better error messages for "Account is locked" - "User or Password incorrect" - "Session transfaired to a other Client" Because the current error messages given are really long and confusing for some users.

**Comment by akallabeth** [comment]
current clients are addressing advanced users, if you want something simpler try remmina.
that being said, simplification of error messages is welcome if there is somebody willing to put some work into that. (but better check back with core developers first to avoid duplicated or wasted efforts)
